config/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Move the `tsconfig` into its own directory so all `subpackages` can eventually read it, and we can write a script to auto-generate new packages.

Reply:
I'm not actually trying to use it with Button in this PR to keep the scope of the PR specific. But we'll get to that!